packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -789,6 +793,9 @@ async function cleanup(projectPath: string): Promise<void> {
   if (await fs.pathExists(path.join(fxConfig, "..", "new.env.default.json"))) {
     await fs.remove(path.join(fxConfig, "..", "new.env.default.json"));
   }
+  if (backupFolder) {
```

Comment:
We need to check if the path of `backupFolder` existed, otherwise fs.remove the file will fail and throw exception?

Reply:
don't need, remove will check. refer: https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove.md

Reply:
793 line is from wenyu, I just not modify it.

Reply:
Curious why we need the check for other files like `new.env.default.json`?

Reply:
ok, got it.